optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -829,6 +829,16 @@ def _cached_file(
             )
         return cached_path
 
+    # TODO: move it to ORTModel (or ORTHubMixin when OptimizedModel is removed)
+    @staticmethod
+    def _infer_file_path(pattern: str, file_name: str, onnx_files: list[Path]):
+        pattern_files = [p for p in onnx_files if re.search(pattern, str(p))]
```

Comment:
Is there a naming rule that can be documented?

Reply:
I'm not aware of any naming rules here, this seemed to me like a repeated pattern of inferring the onnx file path from a pattern with a user given file name, Ideally I want all these file/hub methods to be in ORTHubMixin (or ONNXFileMixin if it makes more sense) but I have already changed too many things in this PR 😅